src/polyfills/events/mobile.js

```diff
@@ -70,13 +71,13 @@
 // on the same element for different events.
 //
 // The current version exposes the following virtual events to jQuery bind methods:
-// "vmouseover vmousedown vmousemove vmouseup vclick vmouseout vmousecancel"
+// "vmouseover vmousedown vmousemove vmouseup vmouseout vmousecancel"
 
 (function( $, window, document, undefined ) {
 
 var dataPropertyName = "virtualMouseBindings",
 	touchTargetPropertyName = "virtualTouchID",
-	virtualEventNames = "vmouseover vmousedown vmousemove vmouseup vclick vmouseout vmousecancel".split( " " ),
+	virtualEventNames = "vmouseover vmousedown vmousemove vmouseup vmouseout vmousecancel".split( " " ),
```

Comment:
Please undo those change, let keep this polyfill as is. It should not get loaded based on our list of supported browser.

Thanks

Reply:
@shiva-sc can you undo this change in this file, just re-add `vclick` thanks.